doc/integrations/duckdb.md

```diff
@@ -15,6 +15,7 @@ myst:
     description lang=en: Use DuckDB from Jupyter using JupySQL
     keywords: jupyter, sql, jupysql, duckdb, plotting
     property=og:locale: en_US
+    property=og:image: https://shorturl.at/hwyCP
```

Comment:
This is the basic JupySQL image. I think we should use the templates designed in this [PR](https://github.com/ploomber/jupysql/pull/398/files) ?

Reply:
Yes! plus this is a shortened url using a 3rd party service, I am looking into replacing it with static images